adobe-esign-connector-product/README.md

```diff
@@ -0,0 +1,134 @@
+
+# Adobe Sign Connector
+
+The Adobe Sign Connector is a project that simplifies the authentication process and enables easy integration and use of Adobe Sign services for signing documents.
+
+[Adobe Sign](https://www.adobe.com/sign.html)
+
+An Adobe Sign account needs to be created to setup and use the connector.
+
+## Adobe Sign account creation
+
+  1. Create a **AdobeSign** Company **Account** OR for Developer Account creation use [Create Developer Account, APIs for custom applications | Acrobat Sign](https://www.adobe.com/sign/developer-form.html) an follow the steps:
```

Comment:
"an follow the steps:" -> we think you mean: "and follow the steps:", isn't it?

Reply:
yes, will fix it asap 😄